youslist.txt

```diff
@@ -18,6 +18,7 @@
 
 ! Exceptions
 @@||ad.smartmediarep.com/NetInsight/$domain=allvod.sbs.co.kr|news.sbs.co.kr|programs.sbs.co.kr
+@@||ssl.pstatic.net/adimg3.search/adpost/js/adpost_show_ads_v2.min.js$script,domain=search.danawa.com/mobile|m.danawa.com
```

Comment:
I think `search.danawa.com/mobile` won't work for domain field.

```suggestion
@@||ssl.pstatic.net/adimg3.search/adpost/js/adpost_show_ads_v2.min.js$script,domain=search.danawa.com|m.danawa.com
```

Reply:
You're correct. I found there's a mistake on that site, after checking it out.